docs/KNOWLEDGE_GUIDE.md

```diff
@@ -15,7 +15,7 @@ You can learn more about the knowledge structure in [Getting Started with Knowle
 ## Accepted Knowledge
 
 > [!IMPORTANT]
-> We are currently only accepting knowledge contributions as a limited private beta and sources will be limited to articles from Wikipedia.
+> We are currently only accepting knowledge contributions as a limited private beta and sources will be limited to those domains listed as approved in our [Knowledge Submissions Review List](https://github.com/instructlab/dev-docs/blob/main/docs/knowledge-submissions-past-wikipedia.md), which includes (but is not necessarily limited to): [Project Gutenberg](https://www.gutenberg.org/), [Wikipedia](https://en.wikipedia.org/wiki/Main_Page), and [Wikisource](https://en.wikisource.org/). See the Knowledge Submissions Review List for further details before you get started with a Knowledge Submission, please and thank you. :D
```

Comment:
```suggestion
> We are currently only accepting knowledge contributions as a limited private beta and sources will be limited to those domains listed as approved in our [Knowledge Submissions Review List](https://github.com/instructlab/dev-docs/blob/main/docs/knowledge-submissions-past-wikipedia.md), which includes (but is not necessarily limited to): [Project Gutenberg](https://www.gutenberg.org/), [Wikipedia](https://en.wikipedia.org/wiki/Main_Page), and [Wikisource](https://en.wikisource.org/). See the Knowledge Submissions Review List for further details before you get started with a Knowledge Submission.
```

`https://github.com/instructlab/dev-docs/blob/main/docs/knowledge-submissions-past-wikipedia.md` is not a good document to link to as it is a design document about a point-in-time (look at the file name) and not actual documentation. We should move the approved list to somewhere else like a doc page in the community repo.

Reply:
^^^ Agree with this, most of the knowledge guides are in the taxonomy repo so I think maybe you can add a copy of the knowledge-submissions-past-wikipedia.md wiki to this repo in this PR,